### DISCUSSION TITLE
Allow custom font name in Settings

### DISCUSSION THREAD

**Discussion by leduyquang753** [monkeytypegame/monkeytype#977] [Ideas] [answered]
There can be a small text box in the *font family* settings section where a custom font family can be entered and then applied.

**Comment by Miodec** [comment, marked as answer]
I've seen some gifs on the discord im not sure if it was you - but maybe replace the input field and the button with just one button that reads 'custom' and shows a popup instead.

**Reply by leduyquang753** [reply]
Yep that was me. I have reimplemented it and I want to make sure this looks good before I open a pull request:

![Screenshot at 19h51 20 17-02-2021](https://user-images.githubusercontent.com/26449060/108207167-f61c2600-7159-11eb-8617-caadc874c770.gif)

**Reply by Miodec** [reply]
Yeah looks good to me!
